Islandora-CLAW/islandora_collection
# Collection header

**GitHub Issue**: Part of https://github.com/Islandora-CLAW/CLAW/issues/716

Other Relevant Links:
- https://github.com/Islandora-CLAW/islandora/pull/68 needs to get merged first.
- https://github.com/Islandora-CLAW/islandora/pull/69, which removes this functionality from the islandora module

# What does this Pull Request do?

Moves the existing collection link header code from the core module to the islandora_collection module, since we're checking for islandora_collection objects.

# What's new?

* The collection link header code @Natkeeran contributed to the islandora module, wrapped up as a service a la https://github.com/Islandora-CLAW/islandora/pull/69
* Functional tests

# How should this be tested?

Create some collections and link them through the use of the "Member Of" field.  Responses to GET and HEAD requests should have a `<collection_url>; rel="collection"` if "Member Of" is filled.  If not, there should be no header.

# Interested parties
@Islandora-CLAW/committers
